apps/person-api/src/main/java/no/nav/registre/testnorge/person/adapter/TpsPersonAdapter.java

```diff
@@ -12,8 +12,9 @@
     private final TpsForvalterenConsumer tpsForvalterenConsumer;
 
     @Override
-    public void createPerson(Person person) {
+    public String createPerson(Person person) {
         //Kommer
+        return null;
```

Comment:
Lag en "TODO" Kommetear og kast en [UnsupportedOperationException](https://docs.oracle.com/javase/7/docs/api/java/lang/UnsupportedOperationException.html) i stede for å retunere null som vil gi ok svar til frontend eller feile i et annet random sted i koden som blir forvirende